### DISCUSSION TITLE
Question marks appearing on prompt style

### DISCUSSION THREAD

**Discussion by BenRacicot** [ohmyzsh/ohmyzsh#10399] [Help] [answered]
Hello,
I have OhMyZSH installed on my original MBP with Solarized Dark profile. Works fine.
<img width="933" alt="og-solarized-dark" src="https://user-images.githubusercontent.com/1815385/140747988-780c31b7-bfe3-489e-82b4-58c87c4405a6.png">

After setting up on my new MBP, Powerline fonts did solves the question mark placeholder character, but  I'm only getting Solarized Dark's background styles.
<img width="799" alt="new-ohmyzsh-install" src="https://user-images.githubusercontent.com/1815385/140748365-af9fc572-a485-4376-bad7-e85a4ad76d12.png">

Can someone help me understand what I am missing to replicate Solarized Dark properly? Thanks in advance.

Note: I've done this several times and read through all docs, also many videos to try'n solve this but am stumped.

**Comment by mcornella** [comment, marked as answer]
See https://github.com/ohmyzsh/ohmyzsh/wiki/FAQ#i-have-a-weird-character-in-my-prompt. In summary: you need to set up a font in your terminal that supports Powerline characters.

**Reply by BenRacicot** [reply]
Hi @mcornella thanks! I've updated the question. The fonts did solve the question mark issue but the theme is still not applied like my original. Do you know of any glaring details I've missed to get the OG theme working?

I dont think Solarized Dark is a Powerline theme, right?

**Reply by mcornella** [reply]
> I dont think Solarized Dark is a Powerline theme, right?

That's just the color scheme of the terminal, it doesn't change what you see in zsh.

The theme in the first screenshot looks like `robbyrussell`, while the theme in the second one looks like `agnoster`. You need to change the OMZ theme to `robbyrussell`.

You can find [more information about Oh My Zsh themes here](https://github.com/ohmyzsh/ohmyzsh/wiki/Themes).

**Reply by BenRacicot** [reply]
Wow, it was right under my nose. Thank you for your time with this, it is very appreciated.
